src/index/src/fulltext_index/tokenizer.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use jieba_rs::Jieba;
+
+use crate::fulltext_index::error::Result;
+use crate::fulltext_index::Bytes;
+
+pub trait Tokenizer: Send {
+    fn tokenize<'a>(&self, text: &'a str) -> Vec<&'a str>;
+}
+
+#[derive(Debug, Default)]
+pub struct EnglishTokenizer;
+
+impl Tokenizer for EnglishTokenizer {
+    fn tokenize<'a>(&self, text: &'a str) -> Vec<&'a str> {
+        text.split(|c: char| !c.is_alphabetic())
```

Comment:
So it will drop the not-alphabetic words such as numbers, operators, etc.?

I think the tokenizer needs some unit test cases.

Reply:
good catch. I have changed it to `is_alphanumeric` which is consistent with `tantivy`